tests/FakeItEasy.Tests.Approval/ApiApproval.cs

```diff
@@ -0,0 +1,19 @@
+namespace FakeItEasy.Tests.Approval
+{
+    using System.Runtime.CompilerServices;
+    using ApprovalTests;
+    using ApprovalTests.Reporters;
+    using NUnit.Framework;
+    using PublicApiGenerator;
+
+    public class ApiApproval
+    {
+        [Test]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]

```

Comment:
Ah, I see. It's because the name of the method is used in the `.approved.txt` file...